src/App.js

```diff
@@ -14,11 +16,32 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    isVisible: false,
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <h1>Goods</h1>
+        {goodsFromServer.length}
+        <div className="wrapper" />
+        {this.state.isVisible && <GoodsList goods={goodsFromServer} />}
+        <button
+          type="button"
+          onClick={(event) => {
+            const elem = event.target;
+
+            elem.style.display = 'none';
```

Comment:
use conditional rendering

Reply:
don't change the DOM directly without necessity, use conditional rendering if you can